src/Vixen.Application/CheckForUpdates.cs

```diff
@@ -50,17 +50,20 @@ private async Task CheckUpdates()
 
 			if (_newVersionAvailable)
 			{
-				labelCurrentVersion.Text = @"Vixen " + _currentVersionType + " " + _latestVersion + @" is now available for download.";
+				labelCurrentVersion.Text = null;

```

Comment:
You should assign string.Empty to this instead of null to avoid future nullable type issues.